themes/bootstrap3/templates/RecordDriver/DefaultRecord/cover.phtml

```diff
@@ -1,16 +1,15 @@
-<?php $alt = $this->link ? trim($this->driver->tryMethod('getTitle') ?? '') : ''; ?>
-<?php if ($this->link && !empty($alt)): ?><a href="<?=$this->escapeHtmlAttr($this->link)?>" class="record-cover-link"><?php endif; ?>
+<?php if ($this->link): ?><a href="<?=$this->escapeHtmlAttr($this->link)?>" class="record-cover-link" tabindex="-1"><?php endif; ?>
```

Comment:
Is the intent here to get rid of alt text on cover images because they are essentially decorative and do not add useful information for a non-sighted user? If so, I think that is probably a reasonable change.

What is the purpose of the tabindex change here?

Reply:
As far as I am aware, the cover appears next to the title, which is also a link. So both the `alt` text and the link are redundant. The link is still useful for mouse users, but for keyboard users it is better to skip it.

Reply:
I wasn't aware that the same code is also used on the record view, so thanks for pointing that out.

Just to be sure: I was under the impression that this was a link *to* the record view. Does the record view link to itself here?

Would it be possible to distinguish the two cases? If not I am fine with dropping this change,

Reply:
In search results, the thumbnail links to the record page. On the record page, the thumbnail links to a larger version of the image, when available. The thumbnail template is generic and receives the href it is supposed to link to from the calling code. We could add another parameter to the template if we need to differentiate behavior in a new way.